.github/workflows/examples.yaml

```diff
@@ -0,0 +1,95 @@
+name: Run examples on push
+
+on:
+  push:
+    branches:
+    - master
+    - develop
+  pull_request:
+    branches:
+    - master
+    - develop
+
+jobs:
+  run-examples:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14.15.4'
+      - name: Cache Node.js modules
+        uses: actions/cache@v2
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+            ${{ runner.OS }}-
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Install python dependencies
+        run: python -m pip install pipenv
+      - run: pipenv sync
+      - name: Install dependencies
+        run: pipenv run yarn install
+      - run: yarn build
+      - run: cd packages/algob && yarn link && cd .. && cd ..
+
+      - name: Setup Network
+        run: sudo apt-get update && sudo apt-get install -y gnupg2 curl software-properties-common
+      - run: curl -O https://releases.algorand.com/key.pub && sudo apt-key add key.pub
+      - run: sudo add-apt-repository "deb [arch=amd64] https://releases.algorand.com/deb/ stable main"
+      - run: sudo apt-get update && sudo apt-get install -y algorand
+      - run: cd infrastructure && make setup-private-net
+
+      - name: Setup algob
+        run: yarn link @algo-builder/algob  && chmod +x ~/.yarn/bin/algob
+
+      - name: Example ASA 
+        run: cd examples/asa && pipenv run yarn algob deploy
+      - run: cd examples/asa && pipenv run yarn algob run scripts/transfer/gold-contract-sc.js
+      - run: cd examples/asa && pipenv run yarn algob run scripts/transfer/gold-delegated-lsig.js
+      - run: cd examples/asa && pipenv run yarn algob run scripts/transfer/gold-to-john.js
+      - run: cd examples/asa && pipenv run yarn algob run scripts/transfer/master-fund-john.js
+      - run: cd examples/asa && pipenv run yarn algob run scripts/transfer/tesla-to-john.js
```

Comment:
do we need to `cd` on each run? i was reading this [https://stackoverflow.com/a/58142276](https://stackoverflow.com/a/58142276). Can we use the same concept here?

Reply:
yes we need it, ok will try this

Reply:
no it doen;t work, it works only for a line under name section, so you have create more names. i think current one is good.

Reply:
ok

Reply:
trying run with `|` , this may work

Reply:
works, thanks